docs/DeveloperGuide.md

```diff
@@ -527,7 +553,52 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     Use case resumes at step 1.
 
 
-*{More to be added}*
+**UC2: Edit a module**
```

Comment:
Hi @WuaaAj it would be good if you could edit your PR to put your use cases into the respective sections 🙇‍♂️
For e.g.
- _edit a module_ -> database features
- _remove all module from a specific semester from the academic plan_ -> academic plan features

Reply:
Okay👌